packages/shared/src/permissions/permissions.utils.ts

```diff
@@ -0,0 +1,46 @@
+import { Permission } from "back-end/types/organization";
+
```

Comment:
TODO: The globalPermissions check and the projectPermissions check have some duplicate logic - figure out a way to remove repeated code.